bin/src/Main.purs

```diff
@@ -536,6 +554,10 @@ main = do
             env' <- runSpago env (mkBuildEnv buildArgs dependencies)
             let options = { depsOnly: true, pursArgs: List.toUnfoldable args.pursArgs, jsonErrors: false }
             runSpago env' (Build.run options)
+          Uninstall { packagesToRemove, selectedPackage, testDeps } -> do
+            { env, fetchOpts } <- mkFetchEnv offline { packages: packagesToRemove, selectedPackage, ensureRanges: false, testDeps: false }
+            let options = { testDeps, dependenciesToRemove: Set.fromFoldable fetchOpts.packages }
```

Comment:
Then I guess we should parse the package names from `packagesToRemove` instead of getting them from `fetchOpts`